apps/crn-server/src/data-providers/contentful/team.data-provider.ts

```diff
@@ -1,11 +1,12 @@
 import {
   LabResponse,
-  ListTeamDataObject,
   TeamCreateDataObject,
   TeamDataObject,
   TeamTool,
   TeamUpdateDataObject,
   TeamMember,
+  TeamReducedDataObject,
+  ListTeamReducedDataObject,
```

Comment:
Why do we have a list team reduced response? Surely this is now the response for all list endpoints?

Reply:
yeah, you're right! I've made the change here [rename ListReducedTeamResponse -> ListTeamResponse](https://github.com/yldio/asap-hub/pull/3929/commits/a2d7da1fbe458a850c637a5f9648b879812eb4db)